probe-rs/src/flashing/mod.rs

```diff
@@ -6,6 +6,44 @@
 //!
 //! It provides a convenient highlevel interface that can flash an ELF, IHEX or BIN file
 //! as well as a lower level block based interface.
+//!
+//!
+//! ## Examples
+//!
+//! ### Flashing a binary
+//!
+//! The easiest way to flash a binary is using the [`download_file`] function,
```

Comment:
Does it need the backticks in the brackets?

Reply:
The backticks change the formatting, the link works with or without.

Reply:
Oki nice :))